pkg/config/apiserver.go

```diff
@@ -228,27 +272,30 @@ func (a *API) deleteRules(req *restful.Request, resp *restful.Response) {
 	writeResponse(http.StatusOK, fmt.Sprintf("Deleted %s", funcPath), nil, resp)
 }
 
-func getRules(store KeyValueStore, path string) (statusCode int, msg string, data *pb.ServiceConfig) {
+func (a *API) getConfig(req *restful.Request, resp *restful.Response, m interface{}) {
+	path := req.Request.URL.Path[len(a.rootPath):]
 	var val string
 	var found bool
 
-	if val, _, found = store.Get(path); !found {
-		return http.StatusNotFound, fmt.Sprintf("no rules for %s", path), nil
+	if val, _, found = a.store.Get(path); !found {
+		writeErrorResponse(http.StatusNotFound, fmt.Sprintf("no rules for %s", path), resp)
```

Comment:
This method is also used for descriptors/adapters but potentially returns an error about "no rules". This might not be a huge deal since the embedded mixer apiserver is likely to go away with the unified/enhanced config server?